README.md

```diff
@@ -194,6 +208,13 @@ Add resources in other languages too :point_down:
 
 ### MongoDB
 - :file_folder: [MongoDB Documentation](https://www.mongodb.com/docs/manual/tutorial/getting-started/)
+
+### MySQL
+- :video_camera: [MySQL Tutorial By Mosh Hamedani](https://youtu.be/7S_tz1z_5bA) 
+
+### PostgreSQL
+- :video_camera: [PostgreSQL Tutorial By Corey Schafer](https://youtu.be/xaWlS9HtWYw)
+
```

Comment:
add MySQL and PostgreSQL in the index too under the backend section, and look at other modules in the index to add them properly.